src/cryptoadvance/specter/managers/device_manager.py

```diff
@@ -125,8 +125,12 @@ def supported_devices_for_chain(self, specter):
                 for device_class in device_classes
                 if device_class.bitcoin_core_support
             ]
+
         if BitcoinCoreWatchOnly in devices:
             devices.remove(BitcoinCoreWatchOnly)
+        # electrum needs to be configured via the "Manual configuration" dialog and excluded from the list
+        if Electrum in devices:
```

Comment:
IMHO, the question is: Why it's in the list in the first place. And if there is a specific way to implement that, why don't we support that. I'll give more thoughts on a comment to the PR.